### DISCUSSION TITLE
Color battery icon red when battery level under 20%

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11258] [Other feature requests]
## The request

Currently Home Assistant displays all battery icons the exact same. I think that when your battery level is getting critically low, you want that to be more obvious at a glance so you can either charge the device or purchase new batteries. Especially since a lot of these Z-Wave sensors use less common battery types like CR123A that aren't always readily available at your local store. 

Many apps and operating systems already do something like this, and the standard usually seems to be when <20%.

![image](https://i.imgur.com/9LQkp9M.jpg)

I think that it definitely makes sense on the Devices areas (listing and details) as those sections are more management/maintenance related in nature. It makes sense on the more-info dialogs as well.

Some might be a little bit more picky about it being in Lovelace, but with the `state_color:` card options they could override whatever default is set and turn the icon coloring on or off as they wish.

## The alternatives

Could do green (high), orange (medium), red (low) but that is overkill IMO. Most people will only care when it needs attention, so just red is fine.

**Comment by bramkragten** [comment]
This is kinda difficult. My phone is indeed fine to show red < 20% because it will die soon. But my sensor might work just fine for 2 months on 20% and I only want to replace it when it is below 5%.

**Comment by SeanPM5** [comment]
That's a very good point, there's not a perfect one size fits all criteria since battery life varies greatly by device. But even if criteria was changed to be below 5% though I think this would still be helpful. 

I rarely pay attention to my sensor battery levels because they last for months. But if I'm scrolling through my Devices list and spot a red icon in the corner of my eye, and see that it's just 3% remaining, that can make the difference between spotting it in time and replacing it or noticing after it's too late.

**Comment by iantrich** [comment]
Setup some alerts on that as well 👍
